### ISSUE TITLE
Mint-X and Mint-Y controls add extra unused blank space on top of toolbar

### ISSUE THREAD

**Issue by Whaym** [linuxmint/mint-themes#169]
![image](https://user-images.githubusercontent.com/18540948/48991117-24376a80-f0f7-11e8-9a01-ac52e81b652c.png)
When I apply any Mint-X or Mint-Y control theme (particularly bad on Mint-X, which nearly doubles the problem) on Cinnamon, some programs (here Firefox) add extra space on the top bar, making tabs not touch the top of the screen when maximized.  This isn't a problem in other popular themes like Adapta-Nokto.

I'm particularly trying to use [Mint-Y-Yltra-Dark](https://cinnamon-spices.linuxmint.com/themes/view/Mint-Y-Yltra-Dark), which I recognize is managed elsewhere, but this is a problem in any Mint-X or Mint-Y related theme.  I can't imagine this is a design choice, as that space is entirely unused and is not universal across programs.

**Comment by JosephMcc** [comment]
What settings are you using. Depending on the settings I have set in Firefox prefs this is totally different. You also aren't giving some basic info like Mint version

**Comment by SehWho** [comment]
![image](https://user-images.githubusercontent.com/18540948/48993006-b728d280-f100-11e8-806c-135ee63508dd.png)
My bad.  I'll note that any of the typical things to play with in about:config and normal settings (drag space, PixelsPerPx, etc) have done what you'd expect but not fixed this particular issue and this same setup works fine with any other theme I've used for controls.  I don't know what specific parts of the Firefox config you'd care about, but as for the related options on the gui, I've currently got drag space off, title bar off, and density at compact.  The space at the top is filled if I set it to touch, but that isn't exactly a solution.
